src/main/webapp/main/LandingPage.js

```diff
@@ -98,11 +101,14 @@ const LandingPage = hh(class LandingPage extends Component{
           return resp;
         });
       }
-    }).catch(() => {});
+    } catch(e) {
+      this.props.hideSpinner()
```

Comment:
Does this have any impact on ErrorHandler? In other words, if there is an error, what takes precedence, this call or that one? Or is there no call to `ErrorHandler` here (which is what I think is true) ... and if so, should this component be wrapped?

Reply:
Additional thought ... should ErrorHandler there be some handing off of the component's spinner to the error handler?

Reply:
As you suspected, we weren't handling any error here so ErrorHandler wouldn't have been triggered. I added its correspondent throw, hiding the spinner before that.